__tests__/app.tests.js

```diff
@@ -52,8 +52,43 @@ describe('GET /api', () => {
                 const docs = body.endpoints;
                 expect(docs).toBeInstanceOf(Object);
                 expect(docs).toEqual(endpoints)
-                console.log(docs);
             });
     });
-    
+});
+
+describe('GET /api/articles/:article_id', () => {
+    test('200: Should return an article object with the correct keys', () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then((response) => {
+                const { article } = response.body;
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
please change to the specific article_ID in the test rather than expect.any(number)